### ISSUE TITLE
RC2: Toolbar item alignment

### ISSUE THREAD

**Issue by dlabrecq** [patternfly/patternfly-react#4339]
Updated to the RC2 packages. Noted an issue with toolbar item alignment.

In the screenshots below, there filter and pagination appear to be indented more than usual. Is this expected? We thought that these controls should align with the table edge?

<img width="1197" alt="Screen Shot 2020-06-04 at 2 58 11 PM" src="https://user-images.githubusercontent.com/17481322/83799469-d698a800-a673-11ea-824d-1119dda699cc.png">

**Comment by redallen** [comment]
Yes, there are supposed to align with the left side and an example of this doing this is in our [Filterable Table Demo](http://patternfly-react.surge.sh/documentation/react/demos/filtertabledemo)

**Comment by dlabrecq** [comment]
The extra padding comes from the `pf-c-toolbar__content` selector of `ToolbarContent`?

Although, this code appears to match the demo layout below?
<img width="552" alt="Screen Shot 2020-06-04 at 10 35 55 PM" src="https://user-images.githubusercontent.com/17481322/83830736-dd470f80-a6b3-11ea-97f2-3d551938eabb.png">

Demo layout
<img width="797" alt="Screen Shot 2020-06-04 at 10 42 30 PM" src="https://user-images.githubusercontent.com/17481322/83831172-d076eb80-a6b4-11ea-832b-c60d090d62bf.png">

**Comment by dlabrecq** [comment]
Discovered these are on `.pf-c-toolbar`, which is adding the extra padding. This may be coming from the Insights chrome?

--pf-c-toolbar--PaddingLeft: var(--pf-global--spacer--lg);
--pf-c-toolbar--md--PaddingRight: var(--pf-global--spacer--md);

**Comment by KendraMar** [comment]
Hey, I think this is related to a Jira I filed a couple weeks back?

https://projects.engineering.redhat.com/browse/RHCLOUD-6552
